### ISSUE TITLE
Greyed out songs in queue when autoplay is turned on

### ISSUE THREAD

**Issue by Maria2clean** [ytmdesktop/ytmdesktop#818]
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [x] I verified that there is no open issue for the same subject.

**Describe the bug**
I cant seem to click any of the choices in the queue when autoplay is turned on.

**To Reproduce**
3. See error

**Expected behavior**
I have seen others' screenshots, they have autoplay turned on and their queue isn't greyed out.  Is this a bug or a feature? I have Youtube Music Premium

**Screenshots**
![YouTube_Music_Desktop_App_XKOZJHhISq](https://user-images.githubusercontent.com/90617752/133078640-4b1fba28-65c3-449d-9150-8e5ef266adad.png)

**Environment (please complete the following information):**
 * YTMDesktop version: 1.13.0
 * OS: Windows
 * OS version: 10
 * Arch: x64
* Installation way: .exe

**Comment by Alipoodle** [comment]
> I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
We're not YouTube Music, this is something to do with them and not us.
To my knowledge (although I am a Non-Premium account) these have never been clickable.

**Comment by Maria2clean** [comment]
> > I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
> 
> We're not YouTube Music, this is something to do with them and not us.
> To my knowledge (although I am a Non-Premium account) these have never been clickable.

My bad, I've never bothered to use the website before and didn't realize these weren't clickable at all. Thanks for the clarification.

**Comment by Maria2clean** [comment]
Turns out Chrome doesn't have any problems when it comes to this, there is also no autoplay button on Chrome, unlike in Brave Browser and in YTM Desktop app.

**Comment by Alipoodle** [comment]
Can you confirm you tested this with the same Song/Track?

Playing an Album and checking queue revealed this.
![image](https://user-images.githubusercontent.com/17199186/133118259-b18aabdc-9aa6-479b-afd1-f6deeab57ff3.png)

But playing a random Video I had played recently did provide me with a queue of songs based upon that.

**Comment by Maria2clean** [comment]
Just got an answer from a YTM Customer Service person and they said that those songs are showing, but aren't technically placed in the queue yet so you cant interact with them. Yes I did use the same track but using Chrome
Here are the differences   (I used the same song which was searched in the same way) 
![Chrome](https://user-images.githubusercontent.com/90617752/133176153-1d841a72-3de6-4ee3-a593-c1f089ba7961.png)
![Brave](https://user-images.githubusercontent.com/90617752/133176164-d12a0e87-53de-41bc-b5cc-f0c2a4675840.png)
![YTMDA](https://user-images.githubusercontent.com/90617752/133182228-58724537-d3f5-422c-bf40-0717d62d5fad.png)

**Comment by NovusTheory** [comment]
Let me explain what's going on here and it's completely YTM doing it. The one with greyed out songs is traditional YouTube autoplay while the other is a radio of sorts. It's extremely weird why they do this and it's inconsistent when they do it. I had to research memory leaks in YTM and the version with greyed out songs tended to be doing it more and causing UI slowness. If I had to guess it's feature flagged or something. I get the non greyed out (radio) in my app
